src/reactviews/common/utils.ts

```diff
@@ -29,21 +25,21 @@ export function formatString(str: string, ...args: any[]): string {
 
 /**
  * Gets the theme type based on the theme passed in
- * @param theme the theme of the react webview
+ * @param themeKind the theme of the react webview
  */
-export function getVscodeThemeType(theme: Theme): string {
-    switch (theme) {
-        case webDarkTheme:
+export function getVscodeThemeType(themeKind: ColorThemeKind): string {
+    switch (themeKind) {
+        case ColorThemeKind.Dark:
             return "vs-dark";
-        case teamsHighContrastTheme:
+        case ColorThemeKind.HighContrast:
             return "hc-black";
         default:
             return "light";
```

Comment:
Do we care about the high-contrast light theme in VS Code?  Or maybe Fluent doesn't have an appropriate mapping for it?

Reply:
I am using this in the table designer monaco editor which doesn't implement the hc-light yet. That's why I am just handling hc-light as light.